src/main/resources/messages_en.properties

```diff
@@ -855,7 +857,16 @@ study.entry.has.samples=Study germplasm has samples.
 study.has.existing.study.entries=Study already has existing study entries.
 invalid.study.entry.property.data.id=Invalid study entry property data id.
 entry.type.does.not.exist=Entry type does not exist. 
-
+study.entry.observation.invalid.length=The max length of value is 255 characters
+study.entry.variable.already-has=Entry: {0} already has the variable: {1}.
+study.entry.variable.not-found=Entry: {0} with variable: {1} not found.
+study.not.has.plot.dataset=Study not has plot dataset.
```

Comment:
@lik-83 improve message: Study does not have plotdataset.

Reply:
Done